styles/home.css

```diff
@@ -0,0 +1,57 @@
+footer {
```

Comment:
If you want to select an html element to style it you should add a class name to it and select it by class name for multiple reasons, specificity is one of them.